src/TogglePassword/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2020-2021 Fabien Potencier
```

Comment:
```suggestion
Copyright (c) 2023-present Fabien Potencier
```

We will need to add that `-present` to the rest of the packages in another PR, but let's do it correctly here :)